resources/serverless/charts/webhook/templates/deployment.yaml

```diff
@@ -47,6 +50,8 @@ spec:
       containers:
         - name: webhook
           volumeMounts:
+            - name: {{ template "fullname" . }}-configuration
```

Comment:
Do we want to have common configuration, for manager and webhook?

Reply:
This is a really good question. I did it consciously but I'm not sure it's better than having separate `configmaps`. I think webhook and controller work together and I can't find a valid use-case to make them separate